### PR TITLE
steve pr

### PR DESCRIPTION
**Pull request by sfrendpax8**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "pax8-frontend-sandbox",
+  "version": "1.0.0",
```
Comment (flintzPax8) [inline package.json:3]: this is all wrong

**Comment by sfrendpax8** [comment]
qwetqwererwq

**Review by stevefrend — COMMENTED** [review]
test1

**Comment by stevefrend** [comment]
shit sucks

**Review by sfrendpax8 — COMMENTED** [review]
asetaset

**Review by sfrendpax8 — COMMENTED** [review]
wqetqtew

**Review by stevefrend — COMMENTED** [review]
reveeee

**Review by stevefrend — COMMENTED** [review]
qwetwqet

**Review by stevefrend — COMMENTED** [review]
qwetteqw

**Review by stevefrend — COMMENTED** [review]
tqweqtwetweq

**Review by stevefrend — COMMENTED** [review]
wqetteqwwtqetwqe

**Review by stevefrend — COMMENTED** [review]
qwettweqtwete

**Review by stevefrend — COMMENTED** [review]
tqewetwtewqtewqtew

**Review by stevefrend — COMMENTED** [review]
qwetteqwtwetweqwte

**Review by stevefrend — COMMENTED** [review]
wqwerqwerqwerqwerasdf

**Review by stevefrend — COMMENTED** [review]
qtwewwererwrew

**Review by stevefrend — APPROVED** [review]

**Pull request merged**